frontend/src/SportsSection.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+class SportsSection extends Component {
+  render() {
+    return (
+      <div className="sport-section">
+      	<h3 className="med-nav-title">Sports</h3>
+        <ul>
+        {
+          this.props.sports.map((sports)=>
+            <li><a href={sports.link}>{sports.name}</a></li>
```

Comment:
Where will these props be passed in? I'm assuming index.js, like we do currently, yeah?